### DISCUSSION TITLE
โหวด Contribute design

### DISCUSSION THREAD

**Discussion by mrthiti** [Naga-DAO/naga-mint-workspace#12] [Polls]
จากโพสนี้ #9 ตอนนี้เราได้ Design มีทั้งหมด 4 รูปแบบ เรามาโหวดกันครับว่ารูปแบบไหนจะโดนใจที่สุด
1. https://svelte.dev/
![image](https://user-images.githubusercontent.com/7018802/163779054-65ef4104-7f41-4191-a92a-20685e12112b.png)

2. https://www.lateral-inc.com/about.html
![image](https://user-images.githubusercontent.com/7018802/163779106-9af9ff4e-3fd6-4ce8-b3cb-4ac95e67659e.png)

3. https://www.etsy.com/team 
![image](https://user-images.githubusercontent.com/7018802/163779142-1b28537c-8bcc-4e9e-a902-3bc129fe7d76.png)

4. Kickstarter (version เก่า)
![image](https://user-images.githubusercontent.com/7018802/163779171-71e187c6-1ad8-439e-83a2-68747ece703d.png)

**ปิดโหวด 18:00น. 19/04/2022**

**Comment by mojisejr** [comment]
ขอยาดโหวต 1
